src/models/Prompts.ts

```diff
@@ -16,7 +16,7 @@ Provide the step-by-step solution, here are the tools you have access to:
 - getSegmentDetails: Use this tool exclusively when a user requests the creation of a segment or a table. It requires specific input parameters, including the names and structures of tables retrieved from the getTables tool and a detailed description of the segment.
 - createChart: Use this tool to generate labels, data and type for a chart generation. This tool will have as input a JSON with the complete data that will have to be filtered and provide only the information related to user's request. The chart type will come indicated in the input message, as line, bar or doghnut, otherwise use bar type. It requires specific input parameters, including the names and structures of tables retrieved from the getTables tool and the sqlQuery tool.
 - filterData: Use this tool when the user asks for data filtering. You will always respond with a list of filtered objects based on the original list, based on user's request. 
-
```

Comment:
Need to review this with you, new tool descriptions can be a lot simpler

Reply:
Done in the newer version of the assistant, already talked about it